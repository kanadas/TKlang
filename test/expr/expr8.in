//silnia
let f = (\f n -> if n > 1 then n * f f (n - 1) else n) in f f 10
