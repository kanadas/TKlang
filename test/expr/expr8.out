expr/expr8.in
Type mismatch: [#0] and [(#0)->(#2)]
